laterpay/asset_sources/js/laterpay-post-view.js

```diff
@@ -207,7 +214,15 @@
                         flipTimePass(this);
                     });
             },
-
+            bindAlreadyPurchasedEvents = function() {
+              // handle clicks on already bought link.
+              $o.body
+              .on('click', $o.lp_already_bought, function(e) {
+                e.preventDefault();
+                sendGAEvent( 'Paid Content Identify' );
+                window.location.href = $(this).attr( 'href' );
```

Comment:
@thrijith Did you test with adblocker on?

Reply:
My ad-blocker was on. Will check and verify once to be thorough.

Reply:
Tested with [this](https://chrome.google.com/webstore/detail/adblock-plus/cfhdojbkjhnklbpkdaibdccddilifddb) enabled. Let me know if needed to test with something else.